plugins/frontend/backstage-plugin-github-pull-requests/src/api/GithubPullRequestsApi.ts

```diff
@@ -15,34 +15,36 @@
  */
 
 import { createApiRef } from '@backstage/core-plugin-api';
-import { PullsListResponseData } from '@octokit/types';
-import { PullRequestState } from '../types';
+import { PullRequestState, SearchPullRequestsResponseData } from '../types';
 
 export const githubPullRequestsApiRef = createApiRef<GithubPullRequestsApi>({
   id: 'plugin.githubpullrequests.service',
 });
 
 export type GithubPullRequestsApi = {
   listPullRequests: ({
+    search,
     token,
     owner,
     repo,
+    defaultFilter,
     pageSize,
     page,
     branch,
     state,
     baseUrl,
   }: {
+    search: string;
     token: string;
     owner: string;
     repo: string;
+    defaultFilter: string;
     pageSize?: number;
     page?: number;
     branch?: string;
     state?: PullRequestState;
     baseUrl: string|undefined;
   }) => Promise<{
-    maxTotalItems?: number;
```

Comment:
not used anymore?

Reply:
yes it was a bad value calculate by the api call function it was page multiply by item per page and in the new endpoint to have pull request we have the real total value